pkg/client/clientpool.go

```diff
@@ -6,8 +6,23 @@ import (
 	spqrlog "github.com/pg-sharding/spqr/pkg/spqrlog"
 )
 
+type RouterClient interface {
```

Comment:
Can we choose a better name? there is already struct with this exact name

Reply:
I think `ClientInfo` will suit, because it correspond to the same structure as in protobuf